workers/utils.py

```diff
@@ -319,3 +320,16 @@ def setup_environ() -> None:
     # Apply
     os.environb.clear()
     os.environb.update(env)
+
+FUZZER_CMD_PORT = 7055
+
+class _PausedFuzzers:
+    def __init__(self):
+        requests.get(f'http://127.0.0.1:{FUZZER_CMD_PORT}/pause')
+
+    def __del__(self):
+        requests.get(f'http://127.0.0.1:{FUZZER_CMD_PORT}/resume')
```

Comment:
You don’t know when the destructor will be called.  You should use [context managers](https://book.pythontips.com/en/latest/context_managers.html) instead.

Reply:
Oh right I thought context managers used init/del and hadn't known of enter/exit, it's my first time implementing one, so sorry for the mistake :sweat_smile: